<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;QUANTIDADE DE ELEMENTOS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTamanho">
       <property name="styleSheet">
        <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>790</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;SÁVIO REIS DE LIMA GONÇALVES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS GERADOS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS DO VETOR ORDENADO SELECTION SORT :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;ATIVIDADE 09&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS DO VETOR ORDENADO INSERTION SORT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOrdenadoInsertion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOrdenadoSelection">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;NÚMERO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÕES:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditExecOrdenSelection">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>210</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;TEMPO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÃO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTempoSS">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditExecOrdenInsertion">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;NÚMERO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÕES:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTempoIS">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>330</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;TEMPO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÃO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGerar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string>GERAR</string>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/gear-solid.svg</normaloff>:/Imagens/gear-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarQtElementos">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/trash-solid.svg</normaloff>:/Imagens/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarSelection">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/trash-solid.svg</normaloff>:/Imagens/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarInsertion">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/trash-solid.svg</normaloff>:/Imagens/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-110</y>
      <width>1151</width>
      <height>1151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Imagens.qrc">:/Imagens/Fundo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarDadosGerados">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/trash-solid.svg</normaloff>:/Imagens/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS DO VETOR ORDENADO BUBBLE SORT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOrdenadoBubble">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditExecOrdenBubble">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTempoBBS">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;NÚMERO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÕES:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>440</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;TEMPO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÃO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarBubble">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>500</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/trash-solid.svg</normaloff>:/Imagens/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS DO VETOR ORDENADO MERGE SORT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOrdenadoMerge">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;NÚMERO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÕES:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditExecOrdenMerge">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>610</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTempoMGS">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>610</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>550</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;TEMPO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÃO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarMerge">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>620</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Imagens.qrc">
      <normaloff>:/Imagens/trash-solid.svg</normaloff>:/Imagens/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS DO VETOR ORDENADO HEAP SORT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOrdenadoHeap">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>720</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>660</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;NÚMERO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÕES:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditExecOrdenHeap">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>720</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTempoHS">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>720</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>660</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;TEMPO &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; color:#ff0000;&quot;&gt;ORDENAÇÃO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_9</zorder>
   <zorder>textEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_18</zorder>
   <zorder>textEditOrdenadoInsertion</zorder>
   <zorder>textEditOrdenadoSelection</zorder>
   <zorder>label_19</zorder>
   <zorder>textEditExecOrdenSelection</zorder>
   <zorder>label_21</zorder>
   <zorder>textEditTempoSS</zorder>
   <zorder>textEditExecOrdenInsertion</zorder>
   <zorder>label_23</zorder>
   <zorder>textEditTempoIS</zorder>
   <zorder>label_22</zorder>
   <zorder>pushButtonGerar</zorder>
   <zorder>pushButtonApagarQtElementos</zorder>
   <zorder>pushButtonApagarSelection</zorder>
   <zorder>pushButtonApagarInsertion</zorder>
   <zorder>pushButtonApagarDadosGerados</zorder>
   <zorder>label_20</zorder>
   <zorder>textEditOrdenadoBubble</zorder>
   <zorder>textEditExecOrdenBubble</zorder>
   <zorder>textEditTempoBBS</zorder>
   <zorder>label_24</zorder>
   <zorder>label_25</zorder>
   <zorder>pushButtonApagarBubble</zorder>
   <zorder>label_26</zorder>
   <zorder>textEditOrdenadoMerge</zorder>
   <zorder>label_27</zorder>
   <zorder>textEditExecOrdenMerge</zorder>
   <zorder>textEditTempoMGS</zorder>
   <zorder>label_28</zorder>
   <zorder>pushButtonApagarMerge</zorder>
   <zorder>label_29</zorder>
   <zorder>textEditOrdenadoHeap</zorder>
   <zorder>label_30</zorder>
   <zorder>textEditExecOrdenHeap</zorder>
   <zorder>textEditTempoHS</zorder>
   <zorder>label_32</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Imagens.qrc"/>
 </resources>
 <connections/>
</ui>
